processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.mergesort.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 1)
--- memsim.hex	2025-12-04 18:14:44.041039317 -0500
+++ memcpu.hex	2025-12-04 18:14:44.037039164 -0500
@@ -15,9 +15,9 @@
 :04000E0018B2A42F51
 :04000F0000B52423F1
 :04001000FFFFFFFFF0
-:0400200000000002DA
+:0400200000000001DB
 :0400220000000001D9
-:0400240000000002D6
+:0400240000000001D7
 :0400260000000001D5
 :040080000000223723
 :04008100FC02021368
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-12-04 18:15:03.366780302 -0500
+++ memcpu.hex	2025-12-04 18:15:03.361780111 -0500
@@ -40,7 +40,8 @@
 :04008E0000012083CA
 :04008F000041011318
 :040090000000806785
-:0400C1000000000338
+:0400C000000000013B
+:0400C100000000013A
 :041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 1)
processing file asmFiles/dual.mergesort.asm (1 of 1)
